src/controllers/accounts/accounts.date.controller.ts

```diff
@@ -141,4 +153,14 @@ const route = async (req: Request, res: Response, next: NextFunction): Promise<v
   }
 };
 
+const updateDateErrorMessage = (errorMessage: string, dataToAppend: string, isFirstError: boolean): string => {
+  let updatedErrorMessage: string = errorMessage;
+  if (!isFirstError) {
+    updatedErrorMessage += " and a " + dataToAppend;
+  } else {
+    updatedErrorMessage += dataToAppend;
+  }
+  return updatedErrorMessage;
+};
+
```

Comment:
I've thought of a better way
```suggestion
let updatedErrorMessage: string = errorMessage;
  if (!isFirstError) {
    updatedErrorMessage += " and a";
  } 
  return updatedErrorMessage + " " + dataToAppend;
```